README.md

```diff
@@ -0,0 +1,17 @@
+# Installation
+### Installlation without docker
+* Clone the repository
+* make sure that Node.js is installed
+* cd into public-idea-list
```

Comment:
Is this still valid? I thought Jessica changed the top level folder name to `app`

Reply:
oh shit, I missed that, I didn't update the README, to add 'app'. I would do that now